leap_c/rl/sac_fop.py

```diff
@@ -153,24 +106,27 @@ def forward(self, obs, mpc_state: MPCBatchedState, deterministic=False):
 
         param = action * self.scale[None, :] + self.loc[None, :]
 
-        param_labels = self.mpc.mpc.param_labels
+        # param_labels = self.mpc.mpc.param_labels
 
-        if action.shape[0] == 1:
-            stats = {
-                param_labels[k]: element for k, element in enumerate(param.squeeze())
-            }
-            self.trainer["trainer"].report_stats(
-                "action",
-                stats,
-                self.trainer["trainer"].state.step,
-            )
+        # if action.shape[0] == 1:
+        #     stats = {
+        #         param_labels[k]: element for k, element in enumerate(param.squeeze())
+        #     }
+        #     self.trainer["trainer"].report_stats(
+        #         "action",
+        #         stats,
+        #         self.trainer["trainer"].state.step,
+        #     )
```

Comment:
I think we should not leave code laying around like this, rather remove it (but please bring back Parameter logging asap :) ).